.github/workflows/fuzzing.yml

```diff
@@ -12,14 +12,14 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        go-version: [ 1.14.x ]
+        go: [ '1.15' ]
```

Comment:
Keep the Go version here at 1.14. There's a [known bug in the go/printer package in Go 1.15](https://github.com/golang/go/issues/40546) which [breaks go-fuzz](https://github.com/dvyukov/go-fuzz/issues/294).

Reply:
:cry: ok